invoicing_app/management/commands/generate_tax_receipts.py

```diff
@@ -166,24 +167,15 @@ def handle(self, **options):
 
         if options['event_id']:
             self.event_id = options['event_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`id` = ' + str(self.event_id)
-            )
+            self.conditional_mask = 'AND `Events`.`id` = ' + str(self.event_id)
 
         if options['user_id']:
             self.user_id = options['user_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`uid` = ' + str(self.user_id)
-            )
+            self.conditional_mask = 'AND `Events`.`uid` = ' + str(self.user_id)
 
-        # If there isn't condition, remove the CONDITION MASK from the query
-        if (not options['user_id']) and (not options['event_id']):
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                ''
-            )
+        # The self.conditional_mask is put into the query
+        self.query = self.query.replace('CONDITION_MASK', '{}')
```

Comment:
Why don't you put the brackets in the query string itself? You could use kwargs for this too
```python
string = "This is a string with {condition_mask}"
formatted_string = string.format(condition_mask="some condition")
```

Reply:
+1 to this

Reply:
You're right! I'll do it!